src/pages/gateway/command-reference.md

```diff
@@ -238,6 +239,177 @@ Workspace ID: 2345678901234567890
 Mesh ID: 12a3b4c5-6d78-4012-3456-7e890fa1bcde
 ```
 
+## aio api-mesh:source:discover
+
+Lists all available sources. Select a source to view its configuration file and copy its content to your clipboard.
+
+<InlineAlert variant="info" slots="text"/>
+
+Sources are prebuilt mesh configuration files that are formatted for a specific combination of sources. This feature is currently in development, see [Create a mesh from a template](./create-mesh.md#create-a-mesh-from-a-template) for more information.
+
+### Usage
+
+```bash
+aio api-mesh:source:discover
+```
+### Flags
+
+`--help` provides information on the specified command.
+
+### Response
+
+```terminal
+[
+  {
+      "name": "Adobe Commerce Compare List",
+      "version": "0.0.2",
+      "description": "Source to get information about Compare list", "author": "Adobe team", "provider",
+      "author": "VZ",
+      "provider": {
+          "name": "Commerce",
+          "handler": {
+            "graphql": {
+              "endpoint": "https://venia.magento.com/graphql/"
+            }
+          },
+          "transforms": [
+            {
+              "rename": {
+                "mode": "bare | wrap",
+                "renames": [
+                  {
+                    "from": {
+                      "type": "Query",
+                      "field": "compareList"
+                    },
+                    "to": {
+                      "type": "Query",
+                      "field": "productCompareList"
+                    }
+                  }
+                ]
+              }
+            },
+            {
+              "filterSchema": {
+                "mode": "bare | wrap",
+                "filters": [
+                  "Query.!category",
+                  "Query.!customerOrders",
+                  "Query.!urlResolver",
+                  "Query.!wishlist"
+                ]
+              }
+            },
+            {
+              "cache": [
```

Comment:
We can skip this cache field this, it doesn't required, but will do the example more clear. The idea here, that we should show that it is "source" configuration